README.md

```diff
@@ -102,14 +102,11 @@ $ echo hello world
 
 hello world
 ~~~~~~~~~~
+
 <!-- CODEBLOCK_END -->
 ```
 
-<!-- CODEBLOCK_END_EXAMPLE2 -->
-
-You can hide the type:value comment from the generated output too:
-
-<!-- CODEBLOCK_START_EXAMPLE3 {"ignore": true} -->
+You can hide the `<type>: <value>` comment from the generated output too:
 
 ```
 <!-- CODEBLOCK_START {"value": "echo hello world", "type": "command", "hideValue": false} -->
```

Comment:
drive by of previous work - shouldn't this be `hideValue: true` to demonstrate the feature?